### DISCUSSION TITLE
Réserve d'énergie supplémentaire ?

### DISCUSSION THREAD

**Discussion by Voisin1234** [Crownicles/Crownicles#2508] [Nouvelles idées]
## Suggestion : Accumulation d’énergie

Bonjour,

Je propose une modification concernant certains mini-événements qui se déroulent durant les trajets. Actuellement, certains événements peuvent restaurer des points d’énergie, mais deviennent inutiles lorsque notre personnage est déjà au maximum de points d’énergie.

### Accumulation d’énergie supplémentaire:
Plutôt que de restaurer des points d’énergie lorsqu’ils sont déjà au maximum, ces mini-événements pourraient permettre d’accumuler de l’énergie supplémentaire.

### Utilisation en combat classé:
Cette énergie accumulée pourrait être utilisée lors d’un combat classé pour avoir plus de points d’énergie disponibles.

### Transparence pour l’adversaire:
L’énergie accumulée devrait être visible par l’adversaire avant un combat. Cela pourrait être indiqué par une émoticône supplémentaire ou simplement être mis à jour sur le profil dans l’énergie existante.

### Limitation pour éviter les abus:
Afin d’éviter les abus, toute l’énergie accumulée serait supprimée une fois arrivé à destination.

Merci d’avoir pris en compte cette suggestion.

Cordialement,

Voisin

**Comment by Voisin1234** [comment]
Cette suggestion ne s’appliquerait pas sur l’île, car l’énergie accumulée est réinitialisée à zéro une fois arrivé à destination. Si vous avez des idées pour résoudre ce problème, n’hésitez pas à les soumettre. J’ai proposé de restaurer l’énergie à zéro à l’arrivée pour simplifier, mais une autre solution pourrait être envisagée.

**Comment by BananePlantain** [comment]
alors pour le coup,gagner de l'énergie même quand t'as toute ton énergie est juste un oubli du vérif dans le code et es pas volontaire

**Reply by Voisin1234** [reply]
Du coup, soit modifier le truc pour le corriger ou alors en profiter pour faire quelque chose avec

**Reply by Ravenclaw-l** [reply]
Oui, ce serait dommage de laisser ça comme ça. Ça fait un mini-événement pour rien. 
Bon, c'est mieux que de tomber sur les gobelets, mais autant que ça serve à quelque chose.
